File: 2st/Yeonwoo/인덱스 바꾸기.py
def solution(my_string, num1, num2):
    answer = ''
    for i, str_ in enumerate(my_string):
        if i==num1:
            answer += my_string[num2]
        elif i==num2:
            answer += my_string[num1]
        else:
            answer += str_
    return answer

File: 3st/Hyeonseo/programmer/인덱스 바꾸기.java
class Solution {
    public String solution(String my_string, int num1, int num2) {
        char[] arr = my_string.toCharArray();
        
        arr[num1] = my_string.charAt(num2);
        arr[num2] = my_string.charAt(num1);
        
        String answer = String.valueOf(arr);
        
        return answer;
    }
}
//level  0